__tests__/app.test.js

```diff
@@ -42,4 +43,23 @@ describe('GET /api/articles', () => {
     test('should return a 200 status code', () => {
         return request(app).get('/api/articles').expect(200);
     });
+    test('returns an array of article objects', () => {
+        return request(app)
+            .get('/api/articles')
+            .then(({ body }) => {
+                expect(body.articles.length).toBeGreaterThan(0);
+                expect(body.articles[0]).toEqual(
+                    expect.objectContaining({
+                        article_id: expect.any(Number),
+                        title: expect.any(String),
+                        topic: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
```

Comment:
the /api/articles endpoint should return articles without the body property, and with an additional property called comment_count